sections/shorting/ManageShort/PositionCard.tsx

```diff
@@ -118,29 +126,27 @@ const PositionCard: FC<PositionCardProps> = ({ short, inputAmount, activeTab })
 
 			if (isCollateralTab) {
 				collateralLockedAmount = isAddCollateralTab
-					? collateralLockedAmount.plus(inputAmount)
-					: collateralLockedAmount.minus(inputAmount);
+					? collateralLockedAmount.add(inputAmount)
+					: collateralLockedAmount.sub(inputAmount);
 			}
 
 			if (isPositionTab) {
 				synthBorrowedAmount = isIncreasePositionTab
-					? synthBorrowedAmount.plus(inputAmount)
-					: synthBorrowedAmount.minus(inputAmount);
+					? synthBorrowedAmount.add(inputAmount)
+					: synthBorrowedAmount.sub(inputAmount);
 			}
 
-			const collateralValue = collateralLockedAmount.multipliedBy(collateralLockedPrice);
+			const collateralValue = collateralLockedAmount.mul(collateralLockedPrice);
 
-			const liquidationPrice = collateralValue.dividedBy(
-				synthBorrowedAmount.multipliedBy(minCollateralRatio)
-			);
+			const liquidationPrice = synthBorrowedAmount.gt(0)
+				? collateralValue.div(synthBorrowedAmount.mul(minCollateralRatio))
+				: wei(0);
 
 			return {
-				collateralLockedAmount: collateralLockedAmount.isNegative()
-					? zeroBN
-					: collateralLockedAmount,
-				collateralValue: collateralValue.isNegative() ? zeroBN : collateralValue,
-				liquidationPrice: liquidationPrice.isNegative() ? zeroBN : liquidationPrice,
-				synthBorrowedAmount: synthBorrowedAmount.isNegative() ? zeroBN : synthBorrowedAmount,
+				collateralLockedAmount: collateralLockedAmount.lt(0) ? zeroBN : collateralLockedAmount,
```

Comment:
do we want to change zeroBN to wei(0) everywhere?  I noticed you did that before. Or maybe just const zeroBN = wei(0) which you might have already done

Reply:
Agreed

Reply:
yea zeroBN and wei(0) are the same thing. I got to the point where I wasn't sure which was better TBH. Not sure if later we want to add a `WEI_0` to the wei lib or something

suggestions?

Reply:
yeah I like that, let's do it